pkg/apis/kops/validation.go

```diff
@@ -377,6 +378,42 @@ func (c *Cluster) Validate(strict bool) error {
 		}
 	}
 
+	// Blacklisted Values
+	err = c.isValidAgainstBlacklist()
+	if err != nil {
+		return fmt.Errorf("Unable to validate against blacklist: %v", err)
+	}
+
+	return nil
+}
+
+type ClusterSpecBlackList struct {
+	KubernetesVersions []string
+}
+
+// Todo Kris - We need to get this into a yaml file that can be read in at compile time.. or do we?
+// If we decide to put this in a model of sorts, we will need to generate the bin data for it..
+// kind of sounds like we are walking down the model path again - so trying things out like this for now
+var CSBlacklist = ClusterSpecBlackList{KubernetesVersions: []string{"1.5.0", "v1.5.0"}}
```

Comment:
Probably yes, but I think it's more important to get it into a dynamically loaded file

I don't know if we need YAML - a simple list of strings might suffice